internal/packager/init_test.go

```diff
@@ -49,14 +49,17 @@ func TestInitFromComposeFileWithFlattenedParams(t *testing.T) {
 version: '3.0'
 services:
   service1:
-    image: image1
     ports:
       - ${ports.service1:-9001}
-
   service2:
-    image: image2
     ports:
-      - ${ports.service2:-9002}
+      - ${ports.service2-9002}
+  service3:
+    ports:
+      - ${ports.service3:?'port is unset or empty in the environment'}
+  service4:
+    ports:
+      - ${ports.service4:?'port is unset or empty in the environment'}
```

Comment:
Shouldn't we test `- ${ports.service4?...` here? I mean without `-` ? As I don't see the purpose of service4 otherwise (just a copy of service3, it does not test anything more than service3 already do).

Reply:
my bad, that 's what I intended to do.